pkg/cli/image/extract/extract.go

```diff
@@ -339,7 +347,7 @@ func (o *ExtractOptions) Run() error {
 					return fmt.Errorf("unable to connect to image repository %s: %v", from.String(), err)
 				}
 
-				srcManifest, location, err := imagemanifest.FirstManifest(ctx, from.Ref, repo, o.FilterOptions.Include)
+				srcManifest, location, err := imagemanifest.FirstManifest(ctx, from.Ref, repo, o.FilterOptions.Include, o.FilterOptions.FilterByOS, o.FilterOptions.IsWildcardFilter())
```

Comment:
how about passing entire `o.FilterOptions`?

Reply:
updated, FirstManifest is now a FilterOptions method